tools/build_paks.py

```diff
@@ -110,16 +117,16 @@
             perforce_content_scene_dir = os.path.realpath(os.path.join(perforce_content_scenes_dir, scene_id))
 
             if spear.path_exists(unreal_project_content_scene_dir):
-                print(f"[SPEAR | build_paks.py] File or directory or symlink exists, removing: {unreal_project_content_scene_dir}")
+                spear.log(f"File or directory or symlink exists, removing: {unreal_project_content_scene_dir}")
                 spear.remove_path(unreal_project_content_scene_dir)
 
-            print(f"[SPEAR | build_paks.py] Creating symlink: {unreal_project_content_scene_dir} -> {perforce_content_scene_dir}")
+            spear.log(f"Creating symlink: {unreal_project_content_scene_dir} -> {perforce_content_scene_dir}")
             os.symlink(perforce_content_scene_dir, unreal_project_content_scene_dir)
 
-        # Now that we have created a symlink, our Unreal project should contain exactly two scenes: starter_content_0000 and scene_id
+        # Now that we have created a symlink, our Unreal project should contain exactly two scenes: debug_0000 and scene_id
         ignore_names = [".DS_Store"]
         unreal_project_scenes = { os.path.basename(x) for x in os.listdir(unreal_project_content_scenes_dir) if x not in ignore_names }
-        assert unreal_project_scenes == {"starter_content_0000", scene_id}
+        assert unreal_project_scenes == {"debug_0000", scene_id}
 
         # see https://docs.unrealengine.com/4.26/en-US/SharingAndReleasing/Deployment/Cooking for more information on these parameters
```

Comment:
Let's update this to https://docs.unrealengine.com/5.2/en-US/cooking-content-in-unreal-engine/ ?

Reply:
Sure